packages/cli/src/config/loadConfig.ts

```diff
@@ -1,28 +1,29 @@
-import { findUpSync } from "find-up";
+import { findUp } from "find-up";
 import path from "path";
-import { ERRORS, MUDError } from "../utils/errors.js";
-
-// TODO require may or may not be needed, finish exploring the various configurations
-import { createRequire } from "module";
-import { fileURLToPath } from "url";
-const require = createRequire(fileURLToPath(import.meta.url));
+import { NotESMConfigError, NotInsideProjectError } from "../utils/errors.js";
 
 // Based on hardhat's config (MIT)
 // https://github.com/NomicFoundation/hardhat/tree/main/packages/hardhat-core
 
-const TS_CONFIG_FILENAME = "mud.config.mts";
+// In order of preference files are checked
+const configFiles = ["mud.config.ts", "mud.config.mts"];
```

Comment:
We could, but it may not be trivial and best left for another PR if needed
And I think we should be opinionated and force people into typescript. We already force ESM, might as well go all the way

Reply:
Ah right, SchemaType enum. That could work in js but not nearly as nice